presto-parser/src/main/java/io/prestosql/sql/parser/SqlParser.java

```diff
@@ -67,6 +69,7 @@ public void syntaxError(Recognizer<?, ?> recognizer, Object offendingSymbol, int
             .ignoredRule(SqlBaseParser.RULE_nonReserved)
             .build();
 
+    private final BiConsumer<SqlBaseLexer, SqlBaseParser> sqlBaseParserInitializer;
```

Comment:
I'd just call this field `initializer` for conciseness.

Reply:
Rebased the PR and renamed the field.